test/modules/lighthouse_service.spec.js

```diff
@@ -568,4 +568,159 @@ describe('lighthouse_service api', () => {
       expect(result).toEqual(expected)
     })
   })
+
+  describe('#generateTestRun', () => {
+    let runId, plateSpecs, addToDart
+
+    beforeEach(() => {
+      mock = jest.spyOn(axios, 'post')
```

Comment:
Don't think you do anything with this `mock` variable?

Reply:
Also, is this the correct way to `spyOn` a function?  I don't know.  I'm just aware that this gets called for every test function here.  Maybe that is a good thing and causes it to reset the count of times the function was called?

Reply:
Can remove both :) `axios.post.mockResolvedValue()` is being used instead to return test specific responses